model.js

```diff
@@ -17,7 +17,33 @@ exports.fetchApiArticlesById = (article_id) => {
       if (rows.length === 0) {
         return Promise.reject({ status: 404, msg: "article doesn't exist" });
       }
-      console.log(rows, "rows from articles table");
       return rows[0];
     });
 };
+
+exports.fetchApiArticles = () => {
+  return db.query(`
+        SELECT articles.article_id, articles.title, articles.topic, articles.author,
+        articles.created_at, articles.votes, articles.article_img_url,
+        CAST(COUNT(comments.comment_id) AS INT) AS comment_count
+      FROM articles
+      LEFT JOIN comments
+      ON articles.article_id = comments.article_id
+      GROUP BY articles.article_id
+      ORDER BY articles.created_at DESC;
+      `).then(({ rows }) => {
+        return rows
+      })
+};
+
+/* Responds with:
```

Comment:
make sure to remove unnecessary comments before pushing so they don't clutter your files